### ISSUE TITLE
missing albumartist translation and missing extra MUSICBRAINZ_ALBUMTYPE

### ISSUE THREAD

**Issue by Andy2244** [beetbox/beets#2764]
### Problem

This [specific release](https://musicbrainz.org/release/f55427b6-e6d9-454c-8510-7423ac1e4060) will not use albumartist translation and only get one of the three listed albumtypes.

The release ends up in a "深澤秀行" folder, instead of "_Soundtracks" because the "MUSICBRAINZ_ALBUMTYPE" is only "spokenword".

The same import via picard 1.4.2 gets the correct english albumartist "Hideyuki Fukasawa" (artist translation option enabled) and all three listed types "album; soundtrack; spokenword" in the "releasetype" field.

### Setup

* OS: linux/docker
* Python version: 2.7.14 
* beets version: 1.4.5
* Turning off plugins made problem go away (yes/no): no

My configuration (output of `beet config`) is:

```yaml
paths:
    default: $albumartist/$album%aunique{} [$year] ($albumtype)/$track - $title
    singleton: _Non-Album/$artist - $title
    comp: _Compilations/$album%aunique{} [$year]/$track - $title
    albumtype:soundtrack: _Soundtracks/$album%aunique{} [$year]/$track - $title
threaded: yes
per_disc_numbering: no
chroma:
    auto: yes
library: /config/musiclibrary.blb

replace:
    ^\.: _
    '[\x00-\x1f]': _
    '[<>:"\?\*\|]': _
    '[\xE8-\xEB]': e
    '[\xEC-\xEF]': i
    '[\xE2-\xE6]': a
    '[\xF2-\xF6]': o
    '[\xF8]': o
    \.$: _
    \s+$: ''
original_date: no

plugins: chroma
directory: /mnt/media/_music/imported/

import:
    write: yes
    copy: no
    move: yes
    resume: ask
    incremental: yes
    quiet_fallback: skip
    timid: no
    log: /config/beet.log
    languages: en
```

**Comment by sampsyo** [comment]
Hello! Here is the alias page for that artist:
https://musicbrainz.org/artist/84c2a1ee-f797-4760-a0d2-b20f608911a6/aliases

Please fill in the type and locale for the aliases you want to use. We only use primary aliases, marked with the language locale you've configured.

Currently, beets does not support multiple album types. Please search the issues here on GitHub for more details.

**Comment by Andy2244** [comment]
I did, but miss-took #2294 for the actual fix of #2200, but seems #2200 is still open than?

**Comment by sampsyo** [comment]
Yep! #2200 is the one, and it’s still open.

Not being too familiar with Picard, I don’t know how its alias selection works! If you find out, it would certainly be interesting to know.
